revdb/src/sqlite_connection.cpp

```diff
@@ -182,6 +182,10 @@ Bool DBConnection_SQLITE::sqlExecute(char *query, DBString *args, int numargs, u
 		}
 
 		int rv = basicExec(newquery, &affectedrows);
+        
+        if (numargs > 0)
+            delete newquery;
```

Comment:
Hmm... serves me right for copy and pasting... although `DBBuffer::grab()` uses `realloc` so I guess `free` is the most appropriate there?